messages/hww.proto

```diff
@@ -129,7 +131,7 @@ message PubResponse {
 message BackupInfo {
   string id = 1;
   uint32 timestamp = 2;
-  uint32 timezone_offset = 3;
+  int32 timezone_offset = 3;
```

Comment:
get rid of it somehow :P

Reply:
I got rid of it. I kept the tag numbers to increase forward/backwards compatibility.